backend/module/chaos/serverexperimentation/server_experimentation.go

```diff
@@ -77,12 +69,30 @@ func (s *Service) transform(config *experimentstore.ExperimentConfig) ([]*experi
 		{
 			Id:    "target",
 			Label: "Target",
-			Value: &experimentation.Property_StringValue{StringValue: fmt.Sprintf("\"%s\" ➡️ \"%s\"", downstream, upstream)},
+			Value: &experimentation.Property_StringValue{StringValue: fmt.Sprintf("%s ➡️ %s", downstream, upstream)},
 		},
 		{
 			Id:    "fault_types",
 			Label: "Fault Types",
-			Value: &experimentation.Property_StringValue{StringValue: description},
+			Value: &experimentation.Property_StringValue{StringValue: faultsDescription},
 		},
 	}, nil
 }
+
+func experimentConfigToString(experiment *serverexperimentation.TestConfig) (string, error) {
+	if experiment == nil {
+		return "", errors.New("experiment is nil")
+	}
+
+	var description string
+	switch experiment.GetFault().(type) {
+	case *serverexperimentation.TestConfig_Abort:
+		description = "Abort"
```

Comment:
small nit: you could just return here `return "Abort"` & below.

Reply:
fixed